Formula/nu-smv.rb

```diff
@@ -12,10 +19,37 @@ class NuSmv < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "pkg-config" => :build
+  depends_on "readline"
+
+  uses_from_macos "bison" => :build
+  uses_from_macos "flex" => :build
+  uses_from_macos "libxml2"
+  uses_from_macos "ncurses"
+
+  resource "minisat" do
+    # This URL must match what is specified in MiniSat/minisat-default.cmake except
+    # that we download the '.tar.gz' instead of the '.zip' to keep "brew audit" happy
+    url "https://github.com/niklasso/minisat/archive/37dc6c67e2af26379d88ce349eb9c4c6160e8543.tar.gz"
```

Comment:
Can we just download the zip, I'd much rather add an exception in brew audit then add hacks in the install to get around it.

Reply:
My question would be *why* does `brew audit` warn if you try to download a github zip?  Are there maybe concerns about sha256 stability which makes it prefer we always use tarballs instead?  (The prohibition on github "zipballs" has been in place since 2014 but @MikeMcQuaid didn't elaborate the reason in his commit message: https://github.com/Homebrew/brew/commit/1070acfe7ac95de63e736c3005efcdf1ce37b472 )  That's why I just went with its suggestion and switched to using the tarball.

The other thing to consider is how Nu-SMV uses the downloaded file.  If you look at `minisat/CMakeLists.txt` inside the tarball its normal path is:
```
    else()
        message("Downloading MiniSat from ${minisat_url} --> ${PROJECT_BINARY_DIR}/${minisat_zip}")
        file(DOWNLOAD ${minisat_url} "${PROJECT_BINARY_DIR}/${minisat_zip}"
[...]
  COMMAND ${CMAKE_COMMAND} -E tar xzf ${minisat_zip}
```
`cmake -E tar` is basically cmake's built-in bsdtar-like tool (i.e. a `tar` command that uses libarchive internally for its main work)  This in turn means just like bsdtar it can read `.zip`/`.tar.gz`/etc automatically.  So from the perspective of the package's build scripts it doesn't matter at all whether we pass in a tarball or a zipball... and in fact it's downloaded with a `tar xzf`-like command which makes it look at first like it even wants a tarball.... it is just expecting a filename that ends in `.zip` which, admittedly, is pretty ugly.

So between the alternatives of hacking on `brew audit` (I don't see any existing `audit_exceptions` file that would quiet this particular warning) or this renaming-the-tarball exercise I think this is the least-ugly of the two options.

Reply:
The only reason is consistency as far as I know. So allowing exceptions is fine, we just didn't need them yet.

Reply:
> The only reason is consistency as far as I know. So allowing exceptions is fine, we just didn't need them yet.

Agreed.

Reply:
OK so this needs a change in homebrew itself to add an `audit_exceptions` file for this case?